sdk/spring/azure-spring-cloud-autoconfigure/src/main/java/com/azure/spring/cloud/actuate/storage/StorageBlobHealthIndicator.java

```diff
@@ -1,27 +1,26 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.spring.autoconfigure.storage.actuator;
+package com.azure.spring.cloud.actuate.storage;
 
 import com.azure.storage.blob.BlobServiceAsyncClient;
-import com.azure.storage.blob.BlobServiceClientBuilder;
 import com.azure.storage.blob.models.StorageAccountInfo;
 import org.springframework.boot.actuate.health.Health;
 import org.springframework.boot.actuate.health.HealthIndicator;
 
-import static com.azure.spring.autoconfigure.storage.actuator.Constants.NOT_CONFIGURED_STATUS;
-import static com.azure.spring.autoconfigure.storage.actuator.Constants.POLL_TIMEOUT;
-import static com.azure.spring.autoconfigure.storage.actuator.Constants.URL_FIELD;
+import static com.azure.spring.cloud.actuate.storage.Constants.NOT_CONFIGURED_STATUS;
+import static com.azure.spring.cloud.actuate.storage.Constants.POLL_TIMEOUT;
+import static com.azure.spring.cloud.actuate.storage.Constants.URL_FIELD;
 
 /**
  * Health indicator for blob storage.
  */
-public class BlobStorageHealthIndicator implements HealthIndicator {
+public class StorageBlobHealthIndicator implements HealthIndicator {
 
     private final BlobServiceAsyncClient internalClient;
```

Comment:
we should have a consistent naming for this property

Reply:
changed